src/app/components/TransactionsTable/index.tsx

```diff
@@ -76,26 +76,39 @@ export default function TransactionsTable({ transactions }: Props) {
                           ~{tx.totalAmountFiat}
                         </p>
                       </div>
-                      <Disclosure.Button className="block h-0 mt-2 text-gray-500 hover:text-black dark:hover:text-white transition-color duration-200">
-                        <CaretDownIcon
-                          className={`${open ? "rotate-180" : ""} w-5 h-5`}
-                        />
-                      </Disclosure.Button>
+                      {([tx.type && "sent", "sending"].includes(tx.type) ||
+                        (tx.type === "received" && tx.boostagram)) && (
+                        <Disclosure.Button className="block h-0 mt-2 text-gray-500 hover:text-black dark:hover:text-white transition-color duration-200">
+                          <CaretDownIcon
+                            className={`${open ? "rotate-180" : ""} w-5 h-5`}
+                          />
+                        </Disclosure.Button>
+                      )}
                     </div>
                   </div>
                   <Disclosure.Panel>
                     <div className="mt-1 ml-9 text-xs text-gray-600 dark:text-neutral-400">
                       {tx.description}
-                      <p>Fee: {tx.totalFees} sats</p>
+                      {tx.totalFees && <p>Fee: {tx.totalFees} sats</p>}
                       {tx.preimage && (
                         <p className="truncate">Preimage: {tx.preimage}</p>
                       )}
-                      {tx.location ? (
+                      {tx.location && (
                         <a href={tx.location} target="_blank" rel="noreferrer">
                           {tx.location}
                         </a>
-                      ) : (
-                        ""
+                      )}
+                      {tx.boostagram && (
+                        <ul>
+                          <li>app_name: {tx.boostagram.app_name}</li>
```

Comment:
This is very basic and not yer translated. What should we show here from the boostagram-data?

Reply:
I think sender_name and message is the most important one. 

Is there a way we can add the rest in a popup or something? 

raw data in getalby.com
<img width="710" alt="image" src="https://user-images.githubusercontent.com/318/178614607-a55f254f-e1a4-4159-9f08-0825396e9eb2.png">

Reply:
Added the `sender_name`. `message` is already there.  
If we really want to show all the detailed info we could add this later on. I'm unsure for what users all the details are interesting. I feel like we could tackle this with the future transaction designs.
What do you think @dvoroneca ?